src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -1747,27 +1747,6 @@ public void BuildOutsideVisualStudio ()
 			}
 		}
 
-		[Test]
```

Comment:
I don't understand why this test was removed.

Reply:
This test is now redundant, the assert was moved into the new `SupportedOSPlatformVersionErrors` test via the `(minSdkVersion = "19"`, `supportedOSPlatVers = "19")` test parameter.